docs/docs-content/integrations/harbor-edge.md

```diff
@@ -0,0 +1,194 @@
+| `harbor-config.auth.certs` | Specifies the certificate and private key used for authentication with Harbor. The common name or subject alternative name must include the host name `spectro.registry.local` and the Edge cluster's virtual IP address. If you don't provide a certificate or private key, Palette generates a certificate and a private key to use internally within the cluster. You can find the certificate in a secret named `harbor-tls` in the `harbor` namespace. |
+| `harbor-config.service.serviceType` | Specifies the service type for the Harbor service. The default service type is NodePort. |
+| `harbor-config.service.harbor` | Specifies the ports that harbor is exposed on. |
+| `harbor-config.storage` | Specifies the size of the Harbor's persistent volume claim in GB. You can configure persistent volume claims for `registry`, `jobService`, `database`, `redis`, and `trivy`. |
+| `harbor-config.service.metrics.enabled` | Specify whether to enable metrics on Harbor. For more information about the kinds of metrics that are exposed, refer to [Harbor metrics documentation](https://goharbor.io/docs/main/administration/metrics/). |
+
+### Usage
+
+#### Enable Harbor to Protect Against Outage
+You can use Harbor in an Edge cluster that is connected to external networks. Harbor stores all container images downloaded from the internet and future image pulls from the cluster will be from the local harbor registry. If your cluster experiences an internet outage, it can still reboot containers or add new nodes using images stored locally in Harbor. For more information, refer to [Deploy a Cluster with a Local Harbor Registry](../clusters/edge/networking/local-registry.md).
+
+#### Log in to Harbor Web UI
+
+With Harbor enabled on your Edge cluster, you can log in to Harbor via its web UI. By default, the Harbor service accessible as a NodePort-type service. with the HTTPS port being 30003. You can use the following steps to log in to Harbor via the web UI. 
+
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. From the left **Main Menu**, navigate to **Clusters**. Choose your Edge cluster.
+
+3. Navigate to the **Nodes** tab, in the **Private Ips** column, you can find the IP addresses of your Edge hosts. 
+
+4. Open a new tab in your browser and navigate to `https://NODE_IP:30003` and replace `NODE_IP` with any IP address in your cluster. NodePort-type services are exposed on the same port on all nodes in your cluster. If you changed the HTTPS port in the configurations, replace the port with the HTTPS port you used. 
```

Comment:
Lenny, should we call out that the user mush have network access to the network CIDR exposed by the Edge host? I'm just thinking of scenarios where an Edge host is in a separate network from the user, which is probably pretty common.

Reply:
i also thought about calling this out explicitly, but didn't because i felt this would be pretty evident. after some reflection so i will still add it